### PR TITLE
feat(Icons): Adding chevron down, download, email, search, star icon

### PR DESCRIPTION
**Pull request by parasjindal**

### CODE REVIEW

File: packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
```
Comment (alinais) [inline packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx:6]: Do we need to have an outline version of "search" icon? 
CC: @codepretty

File: packages/react/src/themes/teams/components/Icon/svg/icons/download.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import cx from 'classnames'
+import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconClassNames } from '../teamsIconClassNames'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g>
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/download.tsx]: There's no harm in leaving it, but the `g` isn't necessary

File: packages/react/src/themes/teams/components/Icon/svg/icons/email.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import cx from 'classnames'
+import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconClassNames } from '../teamsIconClassNames'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g>
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/email.tsx]: There's no harm in leaving it, but the `g` isn't necessary

File: packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx (hunk 1 above)
Comment (codepretty) [reply packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx:6]: This actually is the outline version. I'll request a filled one, but it shouldn't hold up this PR

File: packages/react/src/themes/teams/components/Icon/svg/icons/email.tsx (hunk 3 above)
Comment (codepretty) [reply packages/react/src/themes/teams/components/Icon/svg/icons/email.tsx]: Same for all other svg files

**Review by codepretty — APPROVED** [review]

**Review by kuzhelov — APPROVED** [review]

**Pull request merged**